src/client/client.go

```diff
@@ -47,6 +47,8 @@ type APIClient struct {
 var (
 	//BlockSize is used internally by PFS blockserver
 	BlockSize = 8 * 1024 * 1024 // 8 Megabytes
+	// MaxBlockSize specifies the maximum block size for any data type
+	MaxBlockSize = 100 * 1024 * 1024 // 100 MB
 	// MaxMsgSize is used to define the GRPC frame size, which we need to be greater than a block
 	MaxMsgSize = 3 * BlockSize
```

Comment:
Oh interesting. That's good to know. I spent some time looking through our `GetFile` code to see why the same problem wasn't happening on the other end. The short of that is it looks like we're using bufio, so we're probably writing in 4MB chunks again. I figured that's why it was working. Good to know that the setting only applies to incoming though.

[I had already made this into a separate PR](https://github.com/pachyderm/pachyderm/pull/1336). I think that's ok ... since really this change should've gone along w the buffered `PutFile` PR. And we can make sure that PR lands before this one does.